.github/workflows/add-pr-to-release-project.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: "add merged PRs to release party project"
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
```

Comment:
This should probably use pull_request_target instead of pull_request, so it runs from main instead of the PR branch and can actually access secrets.  I don't think we want to directly run that embedded script from PRs (especially forks).

It might be nice to package that script up as a docker action, too, but perhaps that's overkill.